src/server.ts

```diff
@@ -44,11 +45,10 @@ const startServer = async () => {
     return reply.sendFile(filename)
   })
 
-  server.get('/userInfo', routeUserPermissions)
-
+  server.get('/user-info', routeUserInfo)
```

Comment:
Was wondering why we would switch to hyphens (apart from search engine optimisation) ?

Reply:
From what I've read, snake-case is recommended over camelCase for endpoint and browser paths. Strictly speaking, we should make them case-insensitive too, so /login gives the same thing as /Login. 

But I'm not super fussed, was just trying to follow conventions.

Reply:
ok, I guess it's a bit of a longer discussion, to think about for future projects. I am all good with`kebab-case` or `snake_case` for end points

But just my thoughts, for maybe other projects:

* I've seen plenty of APIs with camelCase (https://developers.google.com/calendar/v3/reference/, DHIS2)
* Is it better to have one case across all of the code base ? Or use recommended conventions from external sources ?
* I am personally leaning toward just using one type of case 'everywhere' for DB for endPoints, files etc.. (previous restriction to using say `camelCase` uniformly across all areas of project where to do with technological barriers, which dont' exists anymore)

Reply:
>     * I've seen plenty of APIs with camelCase (https://developers.google.com/calendar/v3/reference/, DHIS2)

Fair enough, although a lot of URLs with capitals in them are actually case insensitive. e.g. `https://en.wikipedia.org/wiki/Main_Page` also accepts `https://en.wikipedia.org/wiki/main_page` -- it redirects to the canonical one. 

> I am all good with `kebab-case` or `snake_case` for end points

Oh, I thought kebab-case was called snake-case. My bad.

> Is it better to have one case across all of the code base ? Or use recommended conventions from external sources ?

Recommended conventions for me. If I'm using Python, I'm writing in snake_case. Postgres -- snake_case, for reasons previously discussed. Not that it's relevant here, but whenever using capital letters, you need to be 100% sure that case will be preserved throughout your whole workflow, and that's not always the case (Postgres for example, although I know there's ways round that).

Reply:
> Fair enough, although a lot of URLs with capitals in them are actually case insensitive. e.g. `https://en.wikipedia.org/wiki/Main_Page` also accepts `https://en.wikipedia.org/wiki/main_page` -- it redirects to the canonical one.

I guess for endpoints it's not that important, since its intended for access by professionals, but if you're going to have any capitalised URLs that are public-facing, you _must_ make sure that they're case-insensitive -- people would be annoyed if they got a 404 for not capitalising their URLs properly.

Reply:
>     * I've seen plenty of APIs with camelCase (https://developers.google.com/calendar/v3/reference/

Bad example! The camelCase values in all those urls are variable substitutions (e.g. `calendarId` would be a number) - so the actual URLs _would_ be all lower-case 😆

Reply:
> Bad example! The camelCase values in all those urls are variable substitutions (e.g. calendarId would be a number) - so the actual URLs would be all lower-case 😆

check out `calendarList` <- i should have been more specific

Thanks for the reply, appreciated. I'll still push the idea of uniform case (probably camel), for our future apps.

> I guess for endpoints it's not that important, since its intended for access by professionals, but if you're going to have any capitalised URLs that are public-facing, you must make sure that they're case-insensitive -- people would be annoyed if they got a 404 for not capitalising their URLs properly.

if pps are playing around with URLs (very low % of users), they should expect to get 404 if they miss type. (interesting enough `-`and `_` are also only a `shift` key away : )

I guess if we are using a router that is case sensitive, we will 100% have to commit to kebab or snake